core/src/main/kotlin/io/specmatic/core/examples/server/ExamplesInteractiveServer.kt

```diff
@@ -341,8 +341,8 @@ class ExamplesInteractiveServer(
         }
     }
 
-    class ScenarioFilter(filterName: String = "", filterNotName: String = "", filterClauses: List<String> = emptyList(), private val filterNotClauses: List<String> = emptyList()) {
-        private val filter = filterClauses.joinToString(";")
+    class ScenarioFilter(filterName: String = "", filterNotName: String = "", filterClauses: String = "", private val filterNotClauses: List<String> = emptyList()) {
+        private val filter = filterClauses
         private val filterNot = filterNotClauses.joinToString(";")
```

Comment:
filterNot is not used anymore, similarly filterNotClauses is not used, and a parameter for this seems to exist in ExamplesCommand and TestCommand as well. Please remove these parameters.